web/src/views/ConsentPortal/CompletionView.tsx

```
@@ -0,0 +1,76 @@
+import React from "react";
+
+import { Grid, Typography, useTheme } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import { useSearchParams } from "react-router-dom";
+
+import HomeButton from "@components/HomeButton";
+import { Decision, Error, ErrorDescription, ErrorHint, ErrorURI } from "@constants/SearchParams";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)